src/dbus_fast/validators.py

```diff
@@ -10,7 +10,7 @@
 
 _bus_name_re = re.compile(r"^[A-Za-z_-][A-Za-z0-9_-]*$")
 _path_re = re.compile(r"^[A-Za-z0-9_]+$")
-_element_re = re.compile(r"^[A-Za-z_][A-Za-z0-9_]*$")
+_element_re = re.compile(r"^[A-Za-z_][A-Za-z0-9_ ]*$")
```

Comment:
Can you point out where the spec allows spaces?

It doesn't appear to based on how I'm reading it

https://dbus.freedesktop.org/doc/dbus-specification.html
> Each element must only contain the ASCII characters "[A-Z][a-z][0-9]_"

Reply:
Maybe its OK to accept this change (assuming we also get some tests)

I asked other members to take a look in case there is a security impact I'm not thinking about